data/deep missions.txt

```diff
@@ -0,0 +1,2317 @@
+		personality nemesis
+		system destination
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying
+		system destination
+		fleet "Large Northern Pirates"
+		fleet "Small Northern Pirates"
+	
+	on stopover
+		dialog `When you arrive, Kraz Cybernetics already has the cargo waiting by the spaceport. After a few hours of loading the massive freighters, the spaceport crew gives you the 'all clear' signal, and the Behemoth captains get ready to depart.`
+	on complete
+		"deep convoy" ++
+		payment 800000
+		conversation
+			`When you return to <planet> with the freighters, you are greeted by a Deep Security officer who hands you <payment> and thanks you for accepting the job on such a short notice. "We'll make sure to contact you in the future should we need your help. If you want to help us further at the moment, I suggest you check the job boards in the Deep for mystery delivery and retrieval jobs."`
+				decline
+
+
+
+mission "Deep: Mystery Cubes 0"
+	minor
+	name `Gaining an Edge`
+	description `Land on the designated systems to place the mysterious cubes that Lieutenant Paris has given you, then return to <destination>.`
+	source
+		attributes "deep"
+		near "Zosma" 1 10
+		not attributes "station"
+	destination "Memory"
+	stopover "New Sahara"
+	stopover "Bounty"
+	stopover "Heartland"
+	stopover "New Tibet"
+	stopover "New Greenland"
+	stopover "New Switzerland"
+	stopover "Tundra"
+	stopover "Arabia"
+	cargo "mystery cubes" 1
+	to offer
+		"mystery cube" >= 5
+		"combat rating" > 100
+		or
+			and
+				"deep convoy" == 1
+				random < 30
+			and
+				"deep convoy" == 2
+				random < 60
+			and
+				"deep convoy" == 3
+				random < 90
+	
+	on offer
+		conversation
+			`A Deep Security officer calls your name and approaches you as you enter the spaceport's central market. "Greetings, Captain <last>, I'm Lieutenant Paris. Would you be interested in conducting some more work for the Deep? We are currently in need of a few extra ships to assist in an ongoing situation."`
+			choice
+				`	"I'd love to help. What is the situation?"`
+					goto situation
+				`	"Sorry, but I can't help right now."`
+			
+			`	"Whatever plans you have, I suggest you change them, as this is an urgent situation."`
+			choice
+				`	"Okay, I'll help. What is the situation?"`
+				`	"Then I can't help you. Sorry."`
+					decline
+			
+			label situation
+			`	"Excellent. I'm sure that you are aware that much of Deep Security is currently preoccupied, hence why we have been procuring merchants to escort many of the Deep's convoys. The reason this is the case is that we have been attempting to detain a particularly meddlesome pirate warlord who goes by the name of 'Beelzebub'. From what we can ascertain, Beelzebub has extensive knowledge on the inner workings of Deep Security, which is making tracking him down rather difficult, as he seems to know our every move. That is where you and a few other merchants come in."`
```

Comment:
Recommend changing "procuring merchants" to "recruiting merchants". It has the same basic meaning, but "recruiting" sounds more specific to a captain. "Procuring" would make more sense if they were obtaining ships.